connectors/sources/sharepoint_server.py

```diff
@@ -92,7 +92,12 @@ def __init__(self, configuration):
         self.certificate = self.configuration["ssl_ca"]
         self.ssl_enabled = self.configuration["ssl_enabled"]
         self.retry_count = self.configuration["retry_count"]
-        self.site_collections = self.configuration["site_collections"]
+
+        self.site_collections_path = []
```

Comment:
```suggestion
        self.site_collection_paths = []
```

I'd probably rename it like this to indicate that it's an array